### ISSUE TITLE
Linux .NET Core 3.1 GUI app not running

### ISSUE THREAD

**Issue by Dafaque** [Auburn/FastNoiseLite#114]
```
You must install or update .NET to run this application.

App: ~/Downloads/FastNoiseLiteGUI-Linux64
Architecture: x64
Framework: 'Microsoft.NETCore.App', version '3.1.0' (x64)
.NET location: /usr/share/dotnet

The following frameworks were found:
  7.0.9 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]

Learn about framework resolution:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=3.1.0&arch=x64&rid=manjaro-x64
```
So, required version is 3.1, i've got 3.1, what's wrong?

**Comment by Athsmooth** [comment]
For me it does nothing, I think  am executing it right with chmod +x, idk.

**Comment by Athsmooth** [comment]
```
The following frameworks were found:
  7.0.9 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]
```

Maybe you  should remove this version?

**Comment by Auburn** [comment]
Fixed in [v1.1.0](https://github.com/Auburn/FastNoiseLite/releases/tag/v1.1.0) with new web app
